### ISSUE TITLE
Uncaught TypeError: this.resizeStopped is not a function

### ISSUE THREAD

**Issue by JW-M2RD** [icetee/remote-ftp#221]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.1.0
**System**: Mac OS X 10.11.2
**Thrown From**: [Remote-FTP](https://github.com/mgrenier/remote-ftp) package, v0.7.10
### Stack Trace

Uncaught TypeError: this.resizeStopped is not a function

```
At /Users/MrRD/.atom/packages/Remote-FTP/lib/views/tree-view.js:307

TypeError: this.resizeStopped is not a function
  at TreeView.module.exports.TreeView.resizeHorizontalView (/Users/MrRD/.atom/packages/Remote-FTP/lib/views/tree-view.js:307:16)
  at HTMLDocument.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
  at HTMLDocument.elemData.handle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)

```
### Commands

```
     -3:06.8.0 tree-view:remove (li.file.entry.list-item.selected)
     -2:48.5.0 remote-ftp:connect (ol.tree-view.list-tree.has-collapsable-children.focusable-panel.full-menu)
     -2:42.4.0 remote-ftp:create-sftp-config (atom-pane.pane.active)
  8x -2:27.5.0 core:backspace (atom-text-editor.editor.is-focused)
     -2:04.3.0 core:save (atom-text-editor.editor.is-focused)
     -1:51.6.0 remote-ftp:navigate-to (ol.tree-view.list-tree.has-collapsable-children.focusable-panel.full-menu)
     -1:44.6.0 remote-ftp:connect (atom-pane.pane.active)
     -1:26.4.0 remote-ftp:refresh-selected (span.name.icon.icon-file-directory)
     -1:23.6.0 remote-ftp:sync-with-remote (div.header.list-item)
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ],
    "disabledPackages": [
      "remote-sync"
    ]
  },
  "Remote-FTP": {}
}
```
### Installed Packages

``` coffee
# User
Remote-FTP, v0.7.10
language-bison, v1.0.0
language-lex-flex, v0.1.0
terminal-plus, v0.13.0

# Dev
No dev packages
```

**Comment by lauriys** [comment]
it happens when you try to resize the panel with a right click

**Comment by jpxd** [comment]
Wtf, who even does that? But thanks @Hatsuney, I can reproduce it now...

**Comment by jpxd** [comment]
Fixed in f233635f31a98b73262c3e3aba9805c570c587ef

**Comment by lauriys** [comment]
I did it on accident, lol.
